build-vm

```diff
@@ -358,7 +358,7 @@ vm_img_create() {
 
     mkdir -p "${img%/*}" || cleanup_and_exit 4
     # truncate file to the desired size
-    dd if=/dev/zero of="$img" bs=1M count=0 seek="$size" || cleanup_and_exit 4
+    dd if=/dev/zero of="$img" bs=1M count=0 seek="$size" status=none || cleanup_and_exit 4
```

Comment:
I have a similar patch locally that I didn't push yet, and I used `status=noxfer` to only silence the xfer, not everything else. why none here?

Reply:
what else is there to report?